packages/machine/test/unit/cf-operations/fixture.ts

```diff
@@ -68,3 +58,70 @@ export const TEST_FREE_BALANCE = new cf.utils.FreeBalance(
   100,
   new cf.utils.Nonce(true, 0, 5)
 );
+
+export const TEST_FREE_BALANCE_APP_INTERFACE = cf.utils.FreeBalance.contractInterface(
+  TEST_NETWORK_CONTEXT
+);
+
+export const TEST_FREE_BALANCE_APP_INSTANCE = new cf.app.AppInstance(
+  TEST_NETWORK_CONTEXT,
+  TEST_MULTISIG_ADDRESS,
+  TEST_PARTICIPANTS,
+  TEST_FREE_BALANCE_APP_INTERFACE,
+  cf.utils.FreeBalance.terms(),
+  TEST_FREE_BALANCE.timeout,
+  TEST_FREE_BALANCE.uniqueId
+);
+
+export const TEST_APP_INSTANCE = new cf.app.AppInstance(
+  TEST_NETWORK_CONTEXT,
+  TEST_MULTISIG_ADDRESS,
+  TEST_PARTICIPANTS,
+  TEST_APP_INTERFACE,
+  TEST_TERMS,
+  TEST_TIMEOUT,
+  TEST_APP_UNIQUE_ID
+);
+
+export function constructContractCall(funcSig: string, ...args: any): string {
```

Comment:
Is the string `funcSig` referring to some abi-style signature?

Might be useful to add a couple doc lines to this function to clarify what the args/return are.